### DISCUSSION TITLE
Citing CoverM?

### DISCUSSION THREAD

**Discussion by kevinmyers** [wwood/CoverM#151] [Q&A] [answered]
I'd like to cite CoverM in a manuscript. Is there an article I should use or what do you suggest? Just want to make sure you get credit for this great tool!

**Comment by wwood** [comment, marked as answer]
Hi,

Thanks for the thought - much appreciated. I should really write a paper.

Just citing something like "Woodcroft et al. unpublished (https://github.com/wwood/CoverM)" with a version number sounds reasonable to me.

**Reply by kevinmyers** [reply]
Sounds great. Thanks!

**Reply by AroneyS** [reply]
We have now added a CITATION.cff with a Zenodo doi (also see https://zenodo.org/records/10531254).

**Comment by AroneyS** [comment]
We have now added a CITATION.cff with a Zenodo doi (also see https://zenodo.org/records/10531254).

**Reply by kevinmyers** [reply]
This is great! Thank you!
